lib/helper/WebDriver.js

```diff
@@ -944,8 +936,9 @@ class WebDriver extends Helper {
   async grabValueFrom(locator) {
     const res = await this._locate(locator, true);
     assertElementExists(res, locator);
-
-    return forEachAsync(res, async el => el.getValue());
```

Comment:
method returned array, now it returns array or string.

Others helpers have both variants.
@DavertMik , We should review helpers signature

Reply:
Yeah, I think we need to do that for 3.0 release.
Maybe a better way would be to revert my pull request and introduce a new policy for 3.0